channels/ae.m3u

```diff
@@ -177,3 +177,5 @@ http://livecdnh1.tvanywhere.ae:80/hls/star_world/05.m3u8
 http://51.158.73.206/hls/stream.m3u8
 #EXTINF:-1 tvg-id="Wanasah ARB" tvg-name="Wanasah ARB" tvg-logo="https://i.imgur.com/GmgibYW.jpg" group-title="",Wanasah
 http://livecdnh2.tvanywhere.ae/hls/wanasah/04.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/cqdjHHR.png" group-title="Sport",beIN Sports 1 Arabic
+http://line.protv.cc:8000/live/tNxE4rdouh/ttqs53nnsu/7776.m3u8
```

Comment:
we can’t add this channel to this playlist because it’s the Qatar channel

Reply:
latest changes done